static/js/components/UpdateSourceRedshift.jsx

```diff
@@ -113,12 +150,16 @@ const UpdateSourceRedshift = ({ source }) => {
               <Button
                 color="primary"
                 onClick={() => {
-                  handleSubmit(null);
+                  handleSubmit({ redshift: null, redshift_error: null });
                 }}
                 startIcon={<ClearIcon />}
                 size="large"
                 data-testid="nullifyRedshiftButton"
-                disabled={isSubmitting || source.redshift === null}
+                disabled={
+                  isSubmitting ||
+                  source.redshift === null ||
+                  source.redshift_error === null
```

Comment:
Just making sure: the button is disabled if the redshift error is null? event if the redshift itself has a value?

Reply:
Good catch -- we should definitely allow the error to be null

Reply:
This hinges on a policy decision.  In my opinion, one should not enter a redshift without an error, but others may differ on that.  I'm open to discussion on this.

Reply:
I think for now we should allow it to be nullable, and we can revisit it later if needed

Reply:
Implemented null entry for redshift_error, and added test for same.